scout/server/blueprints/alignviewers/templates/alignviewers/igv_sashimi_viewer.html

```diff
@@ -94,7 +94,7 @@ <h3>{{case}}: Splice junctions</h3>
                                     height: 500,
                                   },
                                 ]
-                              }
+                              },
```

Comment:
I knew it! Sorry! :grimacing:

Reply:
No worries! It looks good now I think:
![Screenshot 2024-08-27 at 14 10 02](https://github.com/user-attachments/assets/b319c90f-00ca-476f-b683-5ee170262017)
but I didn't remember that we had the gene track twice!

Reply:
We should have one for each sample

Reply:
Right, only now it's two samples from the same individual. Oh, well, it looks nice I think so let's not worry!